client/src/app/Constants.ts

```diff
@@ -14,13 +14,25 @@ import {
   ProposedAction,
   Risk,
 } from "@app/api/models";
+export interface IEnvVars {
```

Comment:
I think you're regressing the changes from https://github.com/konveyor/tackle2-ui/pull/440 here? You should be able to remove `IEnvVars` from here, restore the `import { ENV } from "./env";` instead of having `export const ENV` here, and then below do:

```ts
export const APP_BRAND = ENV.PROFILE as BrandType
```

I don't think you need the `|| BrandType.Konveyor` if you do that, since `"konveyor"` is already the default for `PROFILE` provided by `ENV`.